beta-src/src/models/OrderClass.ts

```diff
@@ -122,7 +122,7 @@ export default class OrderClass {
       ),
     ).filter((possibleUnitTerritory) => {
       return (
-        possibleUnitTerritory.id !== this.unit.Territory.coastParent.id &&
+        possibleUnitTerritory.id !== this.unit.Territory.coastParent?.id &&
```

Comment:
Can you please explain why you need to make this change?

Reply:
Without this the code errored and said id of undefined

Reply:
Why did it say it was undefined, do you know what the cause is?

Reply:
<img width="1440" alt="Screen Shot 2022-05-04 at 3 30 50 PM" src="https://user-images.githubusercontent.com/95884853/166835945-fffa21d4-da86-4630-9e60-ee627fc1dd4d.png">

I believe its because coastParent isnt an object on there, but coastParentId is a property. Should I change that to coastParentId?

Reply:
Not sure if you added coastParent as an object on purpose

Reply:
Check the existing model.js (the prototype version) code and see what they did there and then compare it what we have. It may have been refactored/copied wrong.

Reply:
model.js is using coastParentID property in Territory property. When switching it to coastParentId it no longer errors

Reply:
Cool, then we solved the bug the right way.